crates/cairo-addons/src/vm/pythonic_hint.rs

```diff
@@ -260,7 +294,13 @@ impl PythonicHintExecutor {
                 .with_serialize()
                 .with_gen_arg()
                 .build();
-            let full_hint_code = format!("{}\n{}", injected_py_code, hint_code);
+            let mut full_hint_code = format!("{}\n{}", injected_py_code, hint_code);
+            // TEMP: remove `from starkware.cairo.lang.builtins.modulo.mod_builtin_runner import
+            // ModBuiltinRunner` imports from the full hints code.
+            full_hint_code = full_hint_code.replace(
+                "from starkware.cairo.lang.builtins.modulo.mod_builtin_runner import ModBuiltinRunner",
+                "from cairo_addons.vm import ModBuiltinRunner",
+            );
```

Comment:
can't we patch at the python level instead? like when we patch eth types

Reply:
not really, we still want to have the python one available for pythonic runs, i suppose?